kalite/distributed/views.py

```diff
@@ -212,6 +213,29 @@ def search(request):
         'category': category,
     }
 
+def create_superuser(request):
+    if request.method == 'POST':
+        intent = request.REQUEST.get('intent', None)
+        if intent == 'addform':
```

Comment:
I'm opposed to the idea of endpoints with different "intents" like this. Makes them harder to test and reason about. Let the intent be determined by the request method (GET, POST, etc.) only. In this case, I'd say create one endpoint to return form html (though I wonder if it can't be put into a handlebars template?) and another endpoint that is the target of the form and ONLY creates a superuser.